kubernetes/chart/zulip/values.yaml

```diff
@@ -0,0 +1,225 @@
+# Default: ClusterIP, needs an Ingress to be used. On Minikube, use a "NodePort"
+# to circumvent the need for an Ingress.
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  # -- Enable this to use an Ingress to reach the Zulip service.
+  enabled: false
+  # -- Can be used to add custom Ingress annotations.
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    # -- Host for the Ingress. Should be the same as
+    # `zulip.environment.SETTING_EXTERNAL_HOST`.
+    - host: zulip.example.com
+      # -- Serves Zulip root of the chosen host domain.
+      paths:
+      - path: /
+  # -- Set a specific secret to read the TLS certificate from. If you use
+  # cert-manager, it will save the TLS secret here. If you do not, you need to
+  # manually create a secret with your TLS certificate.
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - zulip.example.com
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+autoscaling:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 100
+  targetCPUUtilizationPercentage: 80
+  # targetMemoryUtilizationPercentage: 80
+
+# -- Optionally add a nodeSelector to the Zulip pod, so it runs on a specific
+# node.
+# Ref: https://kubernetes.io/docs/user-guide/node-selection/
+nodeSelector: {}
+
+# -- Tolerations for pod assignment.
+# Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+tolerations: []
+
+# -- Affinity for pod assignment.
+# Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+affinity: {}
+
+
+zulip:
+  # Environment variables based on https://github.com/zulip/docker-zulip/blob/master/docker-compose.yml#L63
+  environment:
+    # -- Disables HTTPS if set to "true".
+    # HTTPS and certificates are managed by the Kubernetes cluster, so
+    # by default it's disabled inside the container
+    DISABLE_HTTPS: true
+    # -- Set SSL certificate generation to self-signed because Kubernetes
+    # manages the client-facing SSL certs.
+    SSL_CERTIFICATE_GENERATION: self-signed
+    # -- Domain Zulip is hosted on.
+    SETTING_EXTERNAL_HOST: zulip.example.com
+    # -- SMTP email password.
+    SECRETS_email_password: '123456789'
+    SETTING_ZULIP_ADMINISTRATOR: 'admin@example.com'
+    SETTING_EMAIL_HOST: ''  # e.g. smtp.example.com
+    SETTING_EMAIL_HOST_USER: 'noreply@example.com'
+    SETTING_EMAIL_PORT: '587'
+    # It seems that the email server needs to use ssl or tls and can't be used without it.
```

Comment:
This comment seems cargo-culted in several places.  I don't believe this is a Zulip limitation in any way.  It's often a limitation of the _sending_ server these days, but nothing in Zulip requires this.

Reply:
I think this came from #287. I removed it.